### ISSUE TITLE
404

### ISSUE THREAD

**Issue by DanXS** [mottosso/docker-usd#1]
Hi Marcus,

Nice one putting this together, I've been trying to do the same thing but getting errors building boost, so thought I'd see if anyone had already put in the effort.

I tried yours but getting a 404 - maybe the file doesn't exist anymore?

--2018-12-15 20:47:35--  http://graphics.pixar.com/usd/files/usd-20160420.tar.gz
Resolving graphics.pixar.com... 139.104.180.102
Connecting to graphics.pixar.com|139.104.180.102|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2018-12-15 20:47:36 ERROR 404: Not Found.

Cheers, Dan

**Comment by mottosso** [comment]
Ah yes, they will likely have moved it by now.

Looking for it, it doesn't look like they provide that kind of download anymore..

- https://graphics.pixar.com/usd/downloads.html

..but rather moved everything to GitHub, here:

- https://github.com/PixarAnimationStudios/USD

Perhaps that's already what you're looking at?

**Comment by DanXS** [comment]
Yes thanks, I’ve managed to build it from that Pixar repo for my Mac and my Linux box (ubuntu). But for some reason I’m struggling with a docker based build, not to worry - I’ll get there
